src/Inspiral5PN.cc

```diff
@@ -291,7 +299,14 @@ Pn5Holder Pn5Carrier::run_Pn5(double t0, double M, double mu, double a, double p
 
                 // estimate the step to the breaking point and multiply by PERCENT_STEP
                 xI = Y_to_xI(a, p, e, Y);
-                p_sep = get_separatrix(a, e, xI);
+                if (enforce_schwarz_sep)
```

Comment:
May need to watch out here, as if xI < 0 then the Kerr separatrix will be larger than the Schwarzchild one and the integrator will walk past the Kerr separatrix.